ts/lib/auth/googleauth.ts

```diff
@@ -353,8 +362,8 @@ export default class GoogleAuth {
    * @param {object=} json The input object.
    * @param {function=} opt_callback Optional callback.
    */
-  public fromJSON(json, opt_callback) {
-    let client = null;
+  public fromJSON(json: any, opt_callback: (err: Error, client?: any) => void) {
```

Comment:
Not that I know of - I think it's pretty much a string or a dynamically typed object :)